pages/location/multiLocationsTemplate.js

```diff
@@ -0,0 +1,26 @@
+window.multiLocationsTemplate = `{
```

Comment:
This might be the one I was looking for. I'm wondering why we are using stringified JSON as a template, wouldn't it be better to have a JSON object instead?

Reply:
It's temporary. you can changed it in object.